scripts/common.js

```diff
@@ -746,5 +648,76 @@ export function getLocaleContextedUrl(urlPathToConvert) {
   return pageUrl;
 }
 
+/**
+ * Formats an array of objects into a single object with key-value pairs.
+ * Each object in the array should have 'name' and 'value' properties.
+ * @param {Array} values - The array of objects to format.
+ * @param {string} values[].name - The name property of the object, which will be used as the key.
+ * @param {any} values[].value - The value property of the object, which will be used as the value.
+ * @returns {Object} The formatted object with key-value pairs.
+ */
+const formatValues = (values = []) => {
+  return values.reduce((acc, { name, value }) => {
+    acc[name] = value;
+    return acc;
+  }, {});
+};
+
+function isPerformanceAllowed() {
+  return checkOneTrustGroup(COOKIE_CONFIGS.PERFORMANCE_COOKIE);
+}
+
+function isFunctionalAllowed() {
+  return checkOneTrustGroup(COOKIE_CONFIGS.FUNCTIONAL_COOKIE);
+}
+
+function isTargetingAllowed() {
+  return checkOneTrustGroup(COOKIE_CONFIGS.TARGETING_COOKIE);
+}
+
+function isSocialAllowed() {
+  return checkOneTrustGroup(COOKIE_CONFIGS.SOCIAL_COOKIE);
+}
+
+const { cookieValues, projectConfig, dealerLocator, searchConfig } = await getConstantValues();
+
+// This data comes from the sharepoint 'constants.xlsx' file
+const COOKIE_CONFIGS = formatValues(cookieValues?.data);
+const PROJECT_CONFIG = formatValues(projectConfig?.data);
```

Comment:
this attribute potentially is not needed/deprecated since the git owner, git repo and the sitemap comes from the Admin API

Reply:
Fixed/deleted :) thanks